components/quick_input/quick_input.tsx

```diff
@@ -116,11 +126,11 @@ export default class QuickInput extends React.PureComponent {
         return this.input;
     };
 
-    setInput = (input) => {
+    setInput = (input: HTMLInputElement) => {
         this.input = input;
     }
 
-    onClear = (e) => {
+    onClear = (e: any) => {
```

Comment:
`e` should be of type `React.MouseEvent<HTMLDivElement>`
```suggestion
    onClear = (e: React.MouseEvent<HTMLDivElement>) => {
```

Reply:
You're right, fixed in b29e36ef80f1